Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
+++ /dev/null	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-# Editor-based HTTP Client requests
-/httpRequests/
Index: .idea/Firemen.iml
===================================================================
diff --git a/.idea/Firemen.iml b/.idea/Firemen.iml
deleted file mode 100644
--- a/.idea/Firemen.iml	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
+++ /dev/null	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="WEB_MODULE" version="4">
-  <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: .idea/modules.xml
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
deleted file mode 100644
--- a/.idea/modules.xml	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
+++ /dev/null	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/Firemen.iml" filepath="$PROJECT_DIR$/.idea/Firemen.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: index.php
===================================================================
diff --git a/index.php b/index.php
deleted file mode 100644
--- a/index.php	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
+++ /dev/null	(revision 67996419222d46d8cd0ffa2a153f9cd1df0fd6a4)
@@ -1,31 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
-    <title>Fireman</title>
-</head>
-<body>
-    <header>
-        <div id="logo_container">
-            <img id="logo" src="" alt=""/>
-            <p>logo</p>
-        </div>
-    </header>
-
-    <section id="container_button">
-       <div id="button_page_01">
-           <img id="img_button" src="" alt=""/>
-       </div>
-    </section>
-
-    <footer>
-
-    </footer>
-</body>
-</html>
-
-
-<? php;
